packages/wdio-allure-reporter/README.md

```diff
@@ -162,8 +172,8 @@ Screenshots can be attached to the report by using the `takeScreenshot` function
 First set `disableWebdriverScreenshotsReporting: false` in reporter options, then add in afterStep hook:
 
 ```js title="wdio.conf.js"
-afterStep: function (test, scenario, { error, duration, passed }) {
-  if (error) {
+afterStep: function (step, scenario, result, context) {
+  if (result.error) {
```

Comment:
Any particular reason for this change? It is the same as the previous version.

Reply:
Well, these are the parameters included as comments in the `wdio.conf.js` file, in the afterStep hook:
```
     * @param {ITestCaseHookParameter} world            world object containing information on pickle and test step
     * @param {Object}                 result           results object containing scenario results
     * @param {boolean}                result.passed    true if scenario has passed
     * @param {string}                 result.error     error stack if scenario failed
     * @param {number}                 result.duration  duration of scenario in milliseconds
     * @param {Object}                 context          Cucumber World object
```

And they differ from the parameters shown in the current documentation for the allure reporter, where we have the following: `afterStep: function (test, scenario, { error, duration, passed })`

Perhaps this is my ignorance, but I find this difference confusing.